### ISSUE TITLE
Save experiment log on program exit (Trac #158)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#162]
If a user closes microdrop while an experiment is running, the log for the currently running experiment is not saved.

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/158
```json
{
    "status": "closed", 
    "changetime": "2015-01-13T08:43:59", 
    "description": "If a user closes microdrop while an experiment is running, the log for the currently running experiment is not saved.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1421138639811282", 
    "component": "microdrop", 
    "summary": "Save experiment log on program exit", 
    "priority": "minor", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2014-10-28T21:04:34", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20150105T18:09:21: cfobel changed milestone from "Control board plugin 1.0" to "Microdrop 1.0"

**Comment by cfobel** [comment]
Trac update at 20150113T08:43:59: 
* ryan commented:

> Fixed in commit changeset 6990ada045eacd5e9cb5999bb9ed7f1e62e460a8.


* ryan changed resolution from "" to "fixed"
* ryan changed status from "new" to "closed"
